docs/filemanagement.rst

```diff
@@ -239,6 +239,14 @@ is downloaded. Just set the `remove_bom` parameter in `GetContentString()` or
 `GetContentFile()` - see `examples/strip_bom_example.py` in the GitHub
 repository for an example.
 
+Abusive files
+-------------
+
+Files identified as `abusive`_ (malware, etc.) are only downloadable by the owner.
+If you see a
+'This file has been identified as malware or spam and cannot be downloaded'
+error, set 'acknowledge_abuse=True' parameter in `GetContentFile()`. By using
```

Comment:
Decided not to add this flag to `GetContentString` for now because it is tiny bit more involved and we have no use for it.

Also didn't add `GetContentIOBuffer` to this doc, because it is not documented at all yet 🙁

Reply:
let's add https://developers.google.com/drive/api/v3/reference/files/get#acknowledgeAbuse

Reply:
@shcheklein I suppose you meant v2 https://developers.google.com/drive/api/v2/reference/files/get#acknowledgeAbuse . But I'm not sure how useful that is, who cares what we use internally.